### DISCUSSION TITLE
draw debug on the viewport

### DISCUSSION THREAD

**Discussion by yopiagung23** [pafuhana1213/KawaiiPhysics#120] [Ideas - 要望・アイデア]
Hi @pafuhana1213  is that possible to get a draw Debug collision for bone, length rate, and all limit shapes into a viewport gameplay  (viewport inside of level map) instead of viewport editor?

it would be helpful to test the interaction of collision in the world object & see directly what the result on viewport level

**Comment by pafuhana1213** [comment]
Hi @yopiagung23 ,
That's a great idea! Perhaps I can achieve that by mimicking FAnimNode_StrideWarping. I'll try that next time.

**Comment by pafuhana1213** [comment]
I've updated now! `a.AnimNode.KawaiiPhysics.Debug 1` may help you and me!
![image](https://github.com/pafuhana1213/KawaiiPhysics/assets/8957600/48acec54-91de-4e4e-b62a-e30e5a838fa4)

Thanks for your idea!
